sunbird-platform-core/common-util/src/main/java/org/sunbird/common/responsecode/ResponseMessage.java

```diff
@@ -437,6 +437,7 @@
     String ERROR_USER_UPDATE_PASSWORD = "User is created but password couldn't be updated.";
     String ERROR_BGMS_MISMATCH = "Mismatch in {0} at row - {1}";
     String ERROR_USER_MIGRATION_FAILED = "User migration failed.";
+    String IDENTIFIER_VALIDATION_FAILED="Valid identifier is not present in List, Valid supported identifiers are ";
```

Comment:
we already have same kind of list validation , can we not reused it.

Reply:
No, they are different sir